README.md

```diff
@@ -1,2 +1,78 @@
 # rust
-Docker image for Rust
+
+This is a Docker image with Rust 1.18 installed.
+
+## How to Use
+
+In order to use this image for your project, edit the `techio.yml` file and add the following lines to your project:
+
+```yaml
+    runner: techio/rust:latest
+```
```

Comment:
I'm not sure what to answer to that, I don't know what is a `buildCommand`, and why you want to use it. Note that `cargo build` is invoked when you use `cargo run` or `cargo test`, so you don't really need to execute it separately.